operator/v1/types_ingress.go

```diff
@@ -632,11 +678,44 @@ type AWSClassicLoadBalancerParameters struct {
 	// +kubebuilder:validation:Format=duration
 	// +optional
 	ConnectionIdleTimeout metav1.Duration `json:"connectionIdleTimeout,omitempty"`
+
+	// subnets specifies the subnets to which the load balancer will
+	// attach. The subnets may be specified by either their
+	// ID or name. The total number of subnets is limited to 10.
+	//
+	// In order for the load balancer to be provisioned with subnets:
+	// * Each subnet must exist.
+	// * Each subnet must be from a different availability zone.
+	// * The load balancer service must be recreated to pick up new values.
```

Comment:
This won't render as a list when you do OC describe, new lines are not preserved once you generate the CRD. Can we rewrite to complete sentences please

Reply:
You can see in #1730 how Miheer fixed the formatting for similarly formatted lists in other API godoc.